src/routes/api/users.js

```diff
@@ -1,7 +1,7 @@
-const mongoose = require("mongoose");
+// const mongoose = require("mongoose");
 const router = require("express").Router();
 const passport = require("passport");
-const User = mongoose.model("User");
+// const User = mongoose.model("User");
 
 router.get("/user", function(req, res, next) {
     User.findById(req.payload.id)
```

Comment:
Please why did you comment line 1 and line 4, I see the User initialized in line 4 being used in line 7. Don't you think that would lead to errors

Reply:
Well, It will lead to error when the route is specifically called. I believe the person working on `GET /user` route should modify ```// const mongoose = require("mongoose");
// const User = mongoose.model("User");``` since we are using Sequelize instead. I'm supposing the route is not in use at least for now.
However, uncommenting these lines does not lead to any errors, please advise.